pkgs/development/tools/richgo/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "richgo-${version}";
+  version = "0.2.8";
+  goPackagePath = "github.com/kyoh86/richgo";
+
+  src = fetchFromGitHub {
+    owner = "kyoh86";
+    repo = "richgo";
+    rev = "v${version}";
+    sha256 = "0kbwl3a2gima23zmahk0jxp7wga91bs927a1rp5xl889ikng1n4j";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Enrich `go test` outputs with text decorations.";
+    homepage = https://github.com/kyoh86/richgo;
+    license = licenses.mit;
```

Comment:
Sorry, I missed this the last time around. It looks like 0.2.8 is licensed under the unlicense, not MIT.

Reply:
(Please leave a comment about that as well so that the next version-bumper pays attention to the potential license switch)

Reply:
Nice catch! Fixed and added a link to commit